deploy

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
```

Comment:
hopefully this script is run via a service so not sure how useful this is unless the service know what to do with it. The group should decide if this feature is needed for the service.

Reply:
I say we keep it for potential testing or manual use.